DeveloperSetupGuide.md

```diff
@@ -47,6 +47,7 @@ This guide will take you through the development environment setup for each of t
 5. [pims (Pharmacy Information Management System)](https://github.com/mcode/pims)
 6. [rems-smart-on-fhir](https://github.com/mcode/rems-smart-on-fhir)
 7. [rems-intermediary](https://github.com/mcode/rems-intermediary.git)
+8. [rems-directory](https://github.com/mcode/rems-directory.git)
```

Comment:
I already made these changes in dev, didn't I?

Reply:
I also put rems-directory before rems-intermediary since the directory is needed by the intermediary.

Reply:
ahh I opened this to main by mistake, thats why